README.md

```diff
@@ -49,29 +49,33 @@ libraryDependencies ++= javaFXModules.map(m =>
 )
 ```
 
-If you're using [Mill](http://www.lihaoyi.com/mill/) (note https://github.com/com-lihaoyi/mill/issues/767):
+If you're using [Mill](http://www.lihaoyi.com/mill/):
 
 ```scala
 object yourProject extends ScalaModule {
-
-  // Determine OS version of JavaFX binaries
-  lazy val osName = System.getProperty("os.name") match {
-    case n if n.startsWith("Linux")   => "linux"
-    case n if n.startsWith("Mac")     => "mac"
-    case n if n.startsWith("Windows") => "win"
-    case _                            => throw new Exception("Unknown platform!")
+  def scalaVersion = "3.0.0-RC3"
+
+  // Customize coursier resolution to discover the OS-specific artifacts required by JavaFX
+  // Note: this requires mill >= 0.9.7 (with pr/775 merged)
+  def resolutionCustomizer = T.task {
+    Some((r: coursier.core.Resolution) =>
+       r.withOsInfo(coursier.core.Activation.Os.fromProperties(sys.props.toMap))
+    )
   }
 
-  // Add dependency on JavaFX libraries, OS dependent
-  val javaFXModules = List("base", "controls", "fxml", "graphics", "media", "swing", "web")
-    .map(m => ivy"org.openjfx:javafx-$m:16;classifier=$osName")
+  // Add dependency on JavaFX libraries
+  val javaFXVersion  = "16-ea+7"
```

Comment:
This is referencing older early access version, can you change to regular release "16" in the rest if the README?